### PR TITLE
Small fixes while reviewing the document

### PR DESCRIPTION
**Pull request by denfromufa**
npython, print(), py36, typo, links, simplified API

### CODE REVIEW

**Comment by denfromufa** [comment]
Good catch, TIL about binfmt_misc. However it likely requires registration like on Windows.

Here is a quick test:

```
(root) dta@ubuntune:~/pythonnet$ mono nPython.exe
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> exit()
(root) dta@ubuntune:~/pythonnet$ nPython.exe
nPython.exe: command not found
```

File: readme.html [hunk 1]
```diff
@@ -115,7 +115,7 @@
               <li><a href="#exceptions">Exception Handling</a></li>
               <li><a href="#arrays">Using Arrays</a></li>
               <li><a href="#collections">Using Collections</a></li>
-              <li><a href="#com">COM Components</a></li>
+              <li><a href="#com">COM Componefnts</a></li>
```
Comment (vmuriart) [inline readme.html]: typo?

File: readme.html [hunk 2]
```diff
@@ -422,8 +422,8 @@
     try:
         raise NullReferenceException("aiieee!")
     except NullReferenceException, e:
```
Comment (vmuriart) [inline readme.html]: `except NullReferenceException as e:` for it to be PY3 compatible

Comment (vmuriart) [inline readme.html]: this one is outside of what you were fixing. Just noticed it while reviewing.

File: readme.html (hunk 1 above)
Comment (denfromufa) [inline readme.html]: yep

File: readme.html (hunk 2 above)
Comment (denfromufa) [inline readme.html]: good catch

**Comment by denfromufa** [comment]
@vmuriart should we defer updating PythonNet to pythonnet?

**Comment by vmuriart** [comment]
I didn't see it as part of these changes. Let's not add them yet till we come closer to a decision. `PythonNet` and `pythonnet` might both be valid at the end.

**Comment by denfromufa** [comment]
@filmor do you agree with merging this?

**Pull request merged**